libs/utils/utils.js

```diff
@@ -940,7 +940,13 @@ async function loadPostLCP(config) {
   const georouting = getMetadata('georouting') || config.geoRouting;
   if (georouting === 'on') {
     const { default: loadGeoRouting } = await import('../features/georoutingv2/georoutingv2.js');
-    await loadGeoRouting(config, createTag, getMetadata, loadBlock, loadStyle);
+    config.geoRoutingActive = await loadGeoRouting(
+      config,
+      createTag,
+      getMetadata,
+      loadBlock,
+      loadStyle,
+    );
   }
```

Comment:
you don't need to make all these changes to the georouting files. 
You're already awaiting a response here . 
Just check if locale-modal-v2 has been added to the DOM right here
document.querySelector('.locale-modal-v2')

Reply:
actually in retrospect I don't even think you need to have any code here. Just add that listener I described in your webapp-prompt file, no?